03-aad.md

```diff
@@ -13,27 +13,35 @@ Following the steps below you will result in an Azure AD configuration that will
 | Cluster Admin Group Membership | Association between the Cluster Admin User(s) and the Cluster Admin Security Group. |
 | _Additional Security Groups_   | _Optional._ A security group (and its memberships) for the other built-in and custom Kubernetes roles you plan on using. |
 
-## OCW Team Note:
-- Define and Save Team name:
-> NOTE: This name will be used for naming custom resource groups and hence [Azure naming restrictions](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules) are applied to the resulting name of the resources. . It is recommeneded to use short and unique environment name with 1-5 characters in length which includes:
->- a through z (lowercase letters)
->- 0 through 9 (numbers)
-> 
-  ```bash
-   export TEAM_NAME=<team_name>
-  ```
+## OCW Teams:
+- Define and Save Team name and location:
+   > NOTE: This name will be used for naming custom resource groups and hence [Azure naming restrictions](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules) are applied to the resulting name of the resources. It is recommeneded to use short and unique environment name with 1-5 characters in length which includes:
+   >- a through z (lowercase letters)
+   >- 0 through 9 (numbers)
+
+   | Team Name | Location |
+   |-----------|----------|
+   | team1 | eastus |
+   | team2 | eastus2 |
+   | team3 | centralus |
+   ```bash
+      export TEAM_NAME=<team_name>
+      export TEAM_LOCATION=<team_location>
+   ```
 
 ## Steps
 
-> :book: The Contoso Bicycle Azure AD team requires all admin access to AKS clusters be security-group based. This applies to the new Secure AKS cluster that is being built for Application ID a0008 under the BU001 business unit. Kubernetes RBAC will be AAD-backed and access granted based on a user's AAD group membership.
+> :book: The Azure AD team requires all admin access to AKS clusters be security-group based. This applies to the new Secure AKS cluster that is being built for Application ID a0008 under the BU001 business unit. Kubernetes RBAC will be AAD-backed and access granted based on a user's AAD group membership.
 
```

Comment:
Is "a0008" and "BU001" actually what will be used?  Is it the same for everyone?

Reply:
These are actually generic placeholder for any organization to identify applications and their responsible business units. These will be same across the teams but we could make it more specific to something like "cse" and "ngsa"

Reply:
given the timing, let's leave as-is